cmd/emctl/inject.go

```diff
@@ -37,7 +50,8 @@ func inject(ctx context.Context, c pb.EventMasterClient) error {
 			if err != nil {
 				return errors.Wrap(err, "add event")
 			}
-			log.Printf("%+v", resp.Id)
+			log.Printf("%v %v %v", resp.ID, dc, topic)
+			time.Sleep(2 * time.Second)
```

Comment:
Why the sleep?

Reply:
If the changes are bunched up too closely together it doesn't render nicely at "reasonable resolutions" in Grafana. It works as expected in `eventmaster`, but this is just a utility subcommand to inject some events to be rendered in Grafana, so I added the sleep.